sdks/universal-router-sdk/src/utils/commandParser.ts

```diff
@@ -25,19 +25,32 @@ export type V3PathItem = {
   readonly fee: number
 }
 
-// Parses UniversalRouter commands
+export interface CommandsDefinition {
+  [key: number]: CommandDefinition
+}
+
+// Parses UniversalRouter V2 commands
 export abstract class CommandParser {
```

Comment:
note I wanted to rename this `UniversalRouterV2CommandParser` but would be a breaking change so :shrug: